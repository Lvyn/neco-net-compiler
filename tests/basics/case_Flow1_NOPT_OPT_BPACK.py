from snakes.nets import *

p1 = 'p1'
net = PetriNet('Net')
net.processes = [p1]

f1 = Place('f1', [dot], tBlackToken)
f1.one_safe = True
f1.process_name = p1
f1.flow_control = True

f2 = Place('f2', [], tBlackToken)
f2.one_safe = True
f2.process_name = p1
f2.flow_control = True

net.add_place(f1)
net.add_place(f2)

transition = Transition('t', Expression('True'))
net.add_transition(transition)

net.add_input('f1', 't', Value(dot))
net.add_output('f2', 't', Value(dot))
