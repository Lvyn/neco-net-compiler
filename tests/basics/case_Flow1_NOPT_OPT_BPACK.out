[{
'f1' : [dot],
'f2' : [],
}, {
'f1' : [],
'f2' : [dot],
}, ]
